partest/src/main/scala/scala/tools/nsc/MainGenericRunner.scala

```diff
@@ -75,10 +75,11 @@ class MainGenericRunner {
 
     val sjsCode = {
       val output = WritableMemVirtualJSFile("partest.js")
-      linker.link(ir, output, logger)
+      linker.link(ir, moduleInitializers = Nil, output, logger)
       ResolvedJSDependency.minimal(output) :: Nil
     }
 
+    // TODO We should probably replace this by an ModuleInitializer, eventually.
```

Comment:
It wasn't harder. I think at the time I didn't want to mess with the partest runner. I'll change it.